### PR TITLE
Typos in description of visit and clear procedures

### PR DESCRIPTION
**Pull request by madphysicist**

### CODE REVIEW

File: tutorial/source/appendix.rst [hunk 1]
```diff
@@ -515,8 +515,8 @@ Fields
       static int
       mytype_traverse(mytype* self, visitproc visit, void* arg)
       {
-          Py_VISIT(self->mt_base);
-          Py_VISIT(self->mt_object_member);
+          Py_VISIT(self->mt_object_member);
```
Comment (llllllllll) [inline tutorial/source/appendix.rst]: what is the whitespace change here, is it a tab?

**Comment by llllllllll** [comment]
Thank you very much,  I just had one question about the whitespace.

File: tutorial/source/appendix.rst (hunk 1 above)
Comment (madphysicist) [reply tutorial/source/appendix.rst]: To be honest, I'm not sure. I did this on mobile, where editing the code directly seemed easier than opening an issue. I'll see the PR on a desktop in ~24 hrs and fix it. In the meantime, feel free to do with it as you wish.

**Comment by madphysicist** [comment]
I was looking for an example of how to properly implement the cyclic garbage collection stuff, and boy am I glad I found your tutorial. I think it speaks very well to the clarity of your writing that I was able to see the typo and the obvious intent behind it.

**Comment by madphysicist** [comment]
@llllllllll, I don't know where the weird characters came from exactly (something to do with the GitHub editor on mobile Chrome), but I fixed it all up now. Should be good to go.

**Comment by llllllllll** [comment]
Thanks again!

**Pull request merged**